### ISSUE TITLE
Mousemap destroys right-click

### ISSUE THREAD

**Issue by kgizdov** [jabyrd3/sublime-ixio#1]
Hi,

ixio package comes with a Default.sublime-mousemap file that includes the following:

```
[{
  "button": "button2","count": "2",
  "press_command": "double_click"
}]
```

On Windows 10, this hijacks the right-click action and sublime parses it as:

```
command: double_click {"event": {"button": 2, "x": 510.5, "y": 473.5}}
```

This is regardless of double or single clicking. The issue could be present in other OSes, but I don't have anything else handy at the moment to test it.

Only workaround that I found was to extract the package and comment out the line in the file. This keeps all other ixio functionality and makes me wonder why include it in the first place.

**Comment by jabyrd3** [comment]
I'll take a look this weekend.

**Comment by kgizdov** [comment]
Further to the issue. It seems after commenting out the above lines, I now get the following errors, but with no details.

```
command: context_menu {"event": {"button": 2, "x": 372.5, "y": 492.5}}
TypeError: String required
TypeError: String required
TypeError: String required
TypeError: String required
TypeError: String required
```

**Comment by jabyrd3** [comment]
Sorry i missed this, cleaning it up right now.

**Comment by jabyrd3** [comment]
fixed in v1.0.3
https://github.com/jabyrd3/sublime-ixio/releases/tag/1.0.3
